integration/testnet/network.go

```diff
@@ -1126,6 +1131,12 @@ func BootstrapNetwork(networkConf NetworkConfig, bootstrapDir string) (*flow.Blo
 		DKGPubKeys:                   dkg.PubKeyShares,
 	}
 
+	for i, _ := range participants {
+		if participants[i].Role == flow.RoleObserverService {
+			// It is simulated now by legacy logic
```

Comment:
This is what we discussed in the meeting today. The logic is being written by Shawn. I just simulate the behavior as if it was an access node.

Since this is temporary, I will send out the review later and I try to write a cleaner solution. Too much confusion and complications increase the changes of rejection of the pull request.